doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,21 @@
+# 1. Record architecture decisions
```

Comment:
Right now it still seems like this is a template and a real ADR, which is confusing to me.

If this is the ADR for using ADRs, then shouldn't the context be something along the lines of "We decided to use ADRs to make it easier for future maintainers to understand [blah blah blah]." and the decision "We add ADRs to the `/doc/adr` subdirectory following the advice in [some link]." and then consequences "Hopefully future maintainers are able to understand why the project is the way it is."?

Reply:
Yes! I was definitely a bit more confused on the purpose of this first ADR, but now its much clearer. Thank you for the notes and these changes have been made!